### PR TITLE
Bump uswds to 1.4.2

### PR DESCRIPTION
**Pull request by ncksllvn**
Upgrading is easy this time 🐳 

Resolves department-of-veterans-affairs/vets.gov-team/issues/5883

[Changelog](https://github.com/18F/web-design-standards/releases/tag/v1.4.2)

> 
> Changed
> 
> Use character units (ch) for text max-width for improved performance with different fonts for paragraph text within usa-content (#1866)
> Removed
> 
> Remove unused polyfills from LICENSE (#2189)
> Remove unneeded pattern attribute from date input (#2200)
> Fixed
> 
> Fix hero component button display
> Fix select input text from overlapping with the icon
> Use pixel-fit icon for the select input dropdown for improved display
> Use correct font family for h6 headings
> Fix select arrow display bug after autofill in Chrome
> Fix duplicate border focus on select input in Firefox
> Fix disappearing focus state on links on hover
> Use caption on table for proper accessibility
> Make telephone number a tel link in the footer for more consistent functionality
> Fix spacing issue in multi-line footer links (#1637)
> Fix style consistency issue on optional and required form labels
> Update table styling with a tighter horizontal space, remove gray background on borderless table heading, and flush left first child on borderless table headings (#2192)
> Reduce CSS specificity in table and search components (#2062)

### CODE REVIEW

File: src/sass/style.scss [hunk 1]
```diff
@@ -38,6 +36,7 @@
 
 @import "~uswds/src/stylesheets/core/grid";
 @import "~uswds/src/stylesheets/core/utilities";
+@import "~uswds/src/stylesheets/core/base";
```
Comment (ncksllvn) [inline src/sass/style.scss]: So it turns out that we did have to do a little more than update our dependencies. USWDS changed their import order, so base.scss needed access to the `focus` mixin from the utilities.

**Review by cehsu — APPROVED** [review]
LGTM. I went through the uswds 1.4.2 changelog to check for breaking changes/visual changes. And I have a question about how we should update these moving forward. Should we also go through and remove any fixes that were made, that the update now addresses? For example, it looks like our style overrides for table captions might now be incorporated in the upgrade, if this is true, should we update our styles? Or create a ticket to do so?
Also, similarly, if there are visual changes, should we add them to our list of things that need to be decided upon: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5618?

**Comment by ncksllvn** [comment]
Thanks @cehsu! That's a great question. I think I vote for separate tickets for removing chunks of code, just so we keep the actual dependency update really simple. 
For documenting visual changes I think it's a great idea to use that issue for tracking.

**Comment by cehsu** [comment]
Sounds good @ncksllvn, should we mention the new tickets in the PR description whenever we do this?

**Comment by ncksllvn** [comment]
@cehsu that also sounds good! Feel free to edit the description or I don't mind updating it either.

**Comment by cehsu** [comment]
Sounds good @ncksllvn, please feel free to include any new tickets you create while reviewing the current release changelog and our codebase in the description, such as the table caption issue, if you decide that needs to be updated eventually for example.

**Comment by ncksllvn** [comment]
That's a great idea, will do!

**Comment by ncksllvn** [comment]
This was merged in along with department-of-veterans-affairs/vets-website/pull/6780

**Pull request closed without merging**